src/qt/consolidateunspentwizardselectdestinationpage.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CONSOLIDATEUNSPENTWIZARDSELECTDESTINATIONPAGE_H
+#define CONSOLIDATEUNSPENTWIZARDSELECTDESTINATIONPAGE_H
+
+#include <QWizard>
+
+namespace Ui {
+    class ConsolidateUnspentWizardSelectDestinationPage;
+}
+
+class ConsolidateUnspentWizardSelectDestinationPage : public QWizardPage
+{
+    Q_OBJECT
+
+public:
+    explicit ConsolidateUnspentWizardSelectDestinationPage(QWidget *parent = nullptr);
+    ~ConsolidateUnspentWizardSelectDestinationPage();
+
+signals:
+    void updateFieldsSignal();
+
+public slots:
+    void SetAddressList(const std::map<QString, QString> addressList);
```

Comment:
I think you meant to pass a reference: 

```suggestion
    void SetAddressList(const std::map<QString, QString>& addressList);
```

Reply:
No. This was a copy on purpose. Currently the addressList that is passed to the signal function in the updateLabels function is a local variable. The application will segfault on the consuming side, because the reference will be invalid when the signal is processed by the consumer.

Reply:
The map is not that big...

Reply:
Oops&mdash;I saw the `const` and my eyes expected to see the `&`.

Reply:
I probably should just take the const qualifier out, since it is silly in this context.